### PR TITLE
mongodb_replicaset: Arbiter change and test additions

### PR DESCRIPTION
**Pull request by rhysmeister**
##### SUMMARY

Fix back-to-front logic in arbiter check and include integration tests for this. Thanks to @netjordan for spotting it.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

mongodb_replicaset

### CODE REVIEW

**Comment by gundalow** [comment]
@netjordan could you please test this PR and let us know if it works.

@rhysmeister Bonus points awarded for adding tests

**Comment by rhysmeister** [comment]
@klingac This should fix the issue you reported: https://github.com/ansible/ansible/issues/64781

If you could spare a little time to test this PR it would help get the change merge.

Cheers,

Rhys

File: test/integration/targets/mongodb_replicaset/tasks/main.yml [hunk 1]
```diff
@@ -493,4 +493,48 @@
...
+      - "'MongoDB Replicaset validation failed. Invalid number of replicaset members.' == invalid_num.msg"
+
+- name: Test invalid arbiter handling 2
+  mongodb_replicaset:
+    login_user: admin
+    login_password: secret
+    login_host: "localhost"
+    login_port: 3001
+    login_database: "admin"
+    arbiter_at_index: 4
```
Comment (klingac) [inline test/integration/targets/mongodb_replicaset/tasks/main.yml]: What if I set other invalid value for `arbiter_at_index`? I mean e.g. '-1'. Shouldn't we create another test for it and add validation fot this type of values?

**Review by klingac — APPROVED** [review]
Integration tests spassed. 
I have already test it on real infra and it is fixed.
Please merge this PR

**Comment by gundalow** [comment]
Thanks, merged.
If you'd like this fixed in the next 2.9 release please create a backport PR which includes a `changelog/fragment` file.

**Pull request merged**